src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingUtils.kt

```diff
@@ -446,3 +443,21 @@ fun Collection<VilkårResultat>.splittOppOmKrysserRegelverksendring(): List<Vilk
             listOf(it)
         }
     }
+
+private fun VilkårResultat.krysserRegelendring() =
+    (periodeFom ?: TIDENES_MORGEN).isBefore(DATO_LOVENDRING_2024) &&
+        (periodeTom ?: TIDENES_ENDE).erSammeEllerEtter(DATO_LOVENDRING_2024)
+
+fun Collection<VilkårResultat>.justerTomTilRiktigLengde(): List<VilkårResultat> =
```

Comment:
Kanskje noe mer beskrivende enn "riktig lengde"?/Hva er riktig lengde? 

Hva skjer om man bruker denne funksjonen på ikke adopsjonssaker? Blir det da fortsatt riktig lengde?

Reply:
Skal fortsatt bli riktig hvis det ikke er adopsjon. Metoden forkorter til maksmalt lovlige lengde dersom man har for lang periode. 

Hmm hva med forkortTilGyldigLengde() eller noe i den dur?